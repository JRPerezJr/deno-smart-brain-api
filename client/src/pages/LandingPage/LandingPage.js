import React from 'react';
import { Container } from 'shards-react';

export default function LandingPage() {
  return (
    <Container>
      <h1>Hello</h1>
    </Container>
  );
}
