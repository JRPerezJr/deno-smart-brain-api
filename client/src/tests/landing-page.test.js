import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../pages/landing-page/landing-page.js';

describe('<LandingPage>', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<LandingPage />);
  });

  it('sanity test', () => {
    expect(true).toBe(true);
  });

  it('renders LandingPage component without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
