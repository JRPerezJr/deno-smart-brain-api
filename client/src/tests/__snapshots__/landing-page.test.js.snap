// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage> renders LandingPage component without crashing 1`] = `ShallowWrapper {}`;
